site/app/models/notebook/AbstractNotebookInput.php

```diff
@@ -1,21 +1,21 @@
 <?php
 
-namespace app\models\gradeable;
+namespace app\models\notebook;
 
 use app\libraries\Core;
 use app\models\AbstractModel;
 
 /**
- * Class AbstractGradeableInput
- * @package app\models\gradeable
+ * Class AbstractNotebookInput
+ * @package app\models\notebook
```

Comment:
Please just delete all of these `@package` annotations. We're not generating phpdocs, and they just replicate the information that the namespace already provides.